test/jdk/ProblemList.txt

```diff
@@ -465,6 +465,9 @@ java/awt/KeyboardFocusmanager/ConsumeNextMnemonicKeyTypedTest/ConsumeNextMnemoni
 java/awt/Window/GetScreenLocation/GetScreenLocationTest.java 8225787 linux-x64
 java/awt/Dialog/MakeWindowAlwaysOnTop/MakeWindowAlwaysOnTop.java 8266243 macosx-aarch64
 java/awt/dnd/BadSerializationTest/BadSerializationTest.java 8277817 linux-x64,windows-x64
+java/awt/dnd/DragSourceMotionListenerTest.java 8225131 windows-all
+java/awt/dnd/RejectDragTest.java 7124259 macosx-all
```

Comment:
I wonder if `RejectDragTest` still fails on macOS. Latest comment in [JDK-7124259](https://bugs.openjdk.org/browse/JDK-7124259) is dated February 2020. The problem could be fixed by now.

Reply:
It still fails on macOS.